build.gradle.kts

```diff
@@ -1,18 +1,22 @@
+import org.jetbrains.changelog.Changelog
```

Comment:
The changes in this file where bringing the plugin up inline with the latest template, which was needed to get it to compile under 2022.3